src/streamlink_cli/argparser.py

```diff
@@ -412,9 +412,7 @@ def build_parser():
         "--quiet",
         action="store_true",
         help="""
-            Hide all log output.
-
-            Alias for --loglevel=none.
+            Suppress all console and log output, and also disable user prompts.
```

Comment:
Is this going to be clear enough to users? I feel like this option is typically used by power users so it shouldn't be a problem but do we need to specify "all" user prompts? Because of the way the sentence is structured someone could think it's all console and log output, but only some user prompts. I know this is pedantic but we get a lot of silly questions so it's worth noting.

Reply:
I think this is fine. "disable user prompts" should be pretty clear that no input will be possible when the program asks for input.